meson.build

```diff
@@ -184,7 +184,8 @@ elif host_machine.system() == 'windows'
   cdata.set('SDL_VIDEO_RENDER_D3D', 1)
   cdata.set('SDL_AUDIO_DRIVER_WINMM', 1)
 #  core_args += '
-  core_ldflags += ['winmm.lib', 'version.lib', 'imm32.lib']
+#  core_ldflags += ['winmm.lib', 'version.lib', 'imm32.lib']
```

Comment:
Do we actually need to keep this commented out line?

Reply:
No. Removed.